docs/src/Masonry.doc.js

```diff
@@ -82,6 +82,11 @@ card(
         type: 'boolean',
         defaultValue: false,
       },
+      {
+        name: 'virtualBoundsGetter',
```

Comment:
A couple of things:

1. I'm never much of a fan of the word `getter` - it's not really a "thing" and usually just a synonym for "function". At that point embedding type information in identifiers tends to be not that useful and makes refactoring harder.
2. I think this is a function because Masonry expects absolute values for the bounds (relative to origin point). Perhaps having callers pass in  `virtualBounds{Top,Bottom}` as a relative distance rather than an absolute position might help simplify this prop. i.e.

```jsx
// current
<Masonry virtualBoundsTop={0} virtualBoundsBottom={1} />
// proposal
<Masonry virtualBoundsTop={1} virtualBoundsBottom={2} />
```